Documentation/ceph-cluster-crd.md

```diff
@@ -672,6 +672,16 @@ spec:
 
 ### External cluster
 
+**The minimum supported Ceph version for the External Cluster is Luminous 12.2.x.**
+
+Here is matrix support:
```

Comment:
Some suggestions:
- "The features available from the external cluster will vary depending on the version of Ceph. The following table shows the minimum version of Ceph for some of the features"
- Sort the table by Ceph version
- "Configure extra CRDs" needs more description. What about this? 
"Configure an object store, shared file system, or NFS resources in the local cluster to connect to the external Ceph cluster"